broker-daemon/index.js

```diff
@@ -234,6 +227,18 @@ class BrokerDaemon {
     this.orderbooks.set(marketName, new Orderbook(marketName, this.relayer, this.store.sublevel(marketName), this.logger))
     return this.orderbooks.get(marketName).initialize()
   }
+
+  async validateEngines () {
+    await Promise.all(Array.from(this.engines, async ([ symbol, engine ]) => {
+      try {
+        this.logger.info(`Validating engine configuration for ${symbol}`)
+        await engine.validateNodeConfig()
+        this.logger.info(`Validated engine configuration for ${symbol}`)
+      } catch (e) {
+        await exponentialBackoff(async () => engine.validateNodeConfig(), 10, 1000, () => console.log(`Validated engine configuration for ${symbol}`))
```

Comment:
remove the anonymous functions in this method signature and instead create them in `exponentialBackoff`

Example: `await exponentialBackoff(engine.validateNodeConfig, 10, 1000,...`

Reply:
@martineehrlich actually, we should be able to remove that async anonymous function altogether? `engine.validateNodeConfig()` is already async